app/assets/stylesheets/spotlight/_featured_browse_categories_block.scss

```diff
@@ -0,0 +1,90 @@
+$featured-browse-category-border-color: #b3b3b3;
+$featured-browse-category-caption-color: #f7f7f7;
+$smallest-featured-browse-category-width: 220px;
+$smallest-featured-browse-category-height: $smallest-featured-browse-category-width;
+$largest-featured-browse-category-height: 255px;
+$no-sidebar-medium-image-width: 300px;
+$no-sidebar-large-image-width: 360px;
+$with-sidebar-medium-image-width: 340px;
+$with-sidebar-large-image-width: $no-sidebar-large-image-width;
+
+.browse-category {
+  background-size: cover;
+  background-position: center;
+  border: 2px solid $featured-browse-category-border-color;
+  border-radius: 7px;
```

Comment:
$featured-browse-category-caption-color could be $gray-lighter (just trying to avoid pure white so the text isn't too dominant on the page)